src/System.Security.Permissions/src/System/Security/Permissions/FileIOPermission.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Security.Permissions
+{
+    public sealed partial class FileIOPermission : System.Security.CodeAccessPermission, System.Security.Permissions.IUnrestrictedPermission
+    {
+        public FileIOPermission(System.Security.Permissions.FileIOPermissionAccess access, string path) { }
+        public FileIOPermission(System.Security.Permissions.FileIOPermissionAccess access, string[] pathList) { }
+        public FileIOPermission(System.Security.Permissions.PermissionState state) { }
+        public System.Security.Permissions.FileIOPermissionAccess AllFiles { get; set; }
+        public System.Security.Permissions.FileIOPermissionAccess AllLocalFiles { get; set; }
+        public void AddPathList(System.Security.Permissions.FileIOPermissionAccess access, string path) { }
+        public void AddPathList(System.Security.Permissions.FileIOPermissionAccess access, string[] pathList) { }
+        public override System.Security.IPermission Copy() { return this; }
+        public override bool Equals(object o) => base.Equals(o);
+        public override int GetHashCode() => base.GetHashCode();
+        public string[] GetPathList(System.Security.Permissions.FileIOPermissionAccess access) { return default(string[]); }
+        public override System.Security.IPermission Intersect(System.Security.IPermission target) { return default(System.Security.IPermission); }
+        public override bool IsSubsetOf(System.Security.IPermission target) { return default(bool); }
+        public bool IsUnrestricted() { return default(bool); }
```

Comment:
Throughout all of these src files, I'd find it easier to understand the code if it didn't use default so much, e.g. if instead of ```return default(bool)``` it used ```return false```, if instead of ```return default(string[])``` it returned ```return null```, etc.